README_hi.md

```diff
@@ -0,0 +1,158 @@
+<p align="center">
+  <br/>
+    <img alt="huggingface_hub library logo" src="https://huggingface.co/datasets/huggingface/documentation-images/raw/main/huggingface_hub.svg" width="376" height="59" style="max-width: 100%;">
+  <br/>
+</p>
+
+<p align="center">
+    <i>Huggingface Hub के लिए आधिकारिक पायथन क्लाइंट।</i>
+</p>
+
+<p align="center">
+    <a href="https://huggingface.co/docs/huggingface_hub/ko/index"><img alt="Documentation" src="https://img.shields.io/website/http/huggingface.co/docs/huggingface_hub/index.svg?down_color=red&down_message=offline&up_message=online&label=doc"></a>
+    <a href="https://github.com/huggingface/huggingface_hub/releases"><img alt="GitHub release" src="https://img.shields.io/github/release/huggingface/huggingface_hub.svg"></a>
+    <a href="https://github.com/huggingface/huggingface_hub"><img alt="PyPi version" src="https://img.shields.io/pypi/pyversions/huggingface_hub.svg"></a>
+    <a href="https://pypi.org/project/huggingface-hub"><img alt="downloads" src="https://static.pepy.tech/badge/huggingface_hub/month"></a>
+    <a href="https://codecov.io/gh/huggingface/huggingface_hub"><img alt="Code coverage" src="https://codecov.io/gh/huggingface/huggingface_hub/branch/main/graph/badge.svg?token=RXP95LE2XL"></a>
+</p>
+
+<h4 align="center">
+    <p>
+        <a href="https://github.com/huggingface/huggingface_hub/blob/main/README.md">English</a> |
+        <a href="https://github.com/huggingface/huggingface_hub/blob/main/README_de.md">Deutsch</a> |
+        <a href="https://github.com/huggingface/huggingface_hub/blob/main/README_ko.md">한국어</a>
+    <p>
```

Comment:
This section adds links to READMEs in other languages. So Hindi should be added to it. Let's keep the same order as in the .yaml files.

```html
        <a href="https://github.com/huggingface/huggingface_hub/blob/main/README.md">English</a> |
        <a href="https://github.com/huggingface/huggingface_hub/blob/main/README_de.md">Deutsch</a> |
        <b>Hindi</b>  | <!--TODO: write Hindi in Hindi -->
        <a href="https://github.com/huggingface/huggingface_hub/blob/main/README_ko.md">한국어</a>
```

---

Also other READMEs have to be updated to also take into account the Hindi version (they should all reference each other). Especially:
- [README.md](https://github.com/huggingface/huggingface_hub/blob/d641072189d13c0747727d28a1077f30de80d2da/README.md?plain=1#L21C1-L23C96)
- [README_de.md](https://github.com/huggingface/huggingface_hub/blob/d641072189d13c0747727d28a1077f30de80d2da/README_de.md?plain=1#L21): in this one, can you please add both Hindi and Korean links (we forgot to update it)
- [README_ko.md](https://github.com/huggingface/huggingface_hub/blob/d641072189d13c0747727d28a1077f30de80d2da/README_ko.md?plain=1#L21C1-L23C96)

In all 4 READMEs, the order, text and links should be the same. Only the highlighted language (`<b>...</b>`) changes. Could you take care of this? Thanks in advance!

Reply:
I have added the required translation links as told in review.I did not add the links for the Hindi Translation as the pull request has not been added to the main branch yet.
@Wauplin Can you please take a look at it?
Thank you,
Aneesh 😄

Reply:
> I have added the required translation links as told in review.I did not add the links for the Hindi Translation as the pull request has not been added to the main branch yet.

@aneeshd27  You can already set them to:

```
<a href="https://github.com/huggingface/huggingface_hub/blob/main/README_hi.md">हिंदी</a>
```

in README.md, README_de.md and README_ko.md. So that when we merge this PR, everything will be correct :)